{
  "deprecated": false,
  "cpeName": "cpe:2.3:a:python:setuptools:58.5.3:*:*:*:*:*:*:*",
  "cpeNameId": "E357AC29-B00E-498E-8079-E3BFCC27F47D",
  "created": "2022-12-29T14:39:02.883",
  "lastModified": "2022-12-29T14:43:10.977",
  "titles": [
    {
      "title": "Python setuptools 58.5.3",
      "lang": "en"
    }
  ],
  "refs": [
    {
      "ref": "https://pypi.org/project/setuptools/#history",
      "type": "Version"
    },
    {
      "ref": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages/",
      "type": "Advisory"
    }
  ]
}
